src/components/Article.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import styled from 'styled-components';
+import { SectionHeader } from 'GlobalStyles';
+
+const Container = styled.article`
+background: white;
+padding-right: 4em; 
+padding-left: 4em; 
+padding-top: 2em; 
+padding-bottom: 2em; 
+display: grid; 
+grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+justify-content: center;
+align-content: center;  
+ `
+
+const ArticleContainer = styled.div`
```

Comment:
when you inspect and use smaller versions of devices this image is wider than the rest 
<img width="574" alt="Screenshot 2022-10-30 at 19 47 24" src="https://user-images.githubusercontent.com/110664183/198896115-93a507c5-b114-43c7-a32f-8cc3e6ea0a7c.png">